ui/StatusQ/src/StatusQ/Controls/StatusTagItem.qml

```diff
@@ -66,51 +67,52 @@ Rectangle {
         }
     }
 
-    width: tagRow.implicitWidth + 2 * d.tagMargins
+    width: (tagRow.implicitWidth + (2 * d.tagMargins))
     height: 30
-    color: d.getTagColor(root.isReadonly)
-    radius: 8
-    Row {
+    background: Rectangle {
+        color: d.getTagColor(root.isReadonly)
+        radius: 8
+    }
+
+    contentItem: RowLayout {
         id: tagRow
         height: parent.height
         anchors.left: parent.left
+        anchors.right: parent.right
         anchors.leftMargin: d.tagMargins
         anchors.rightMargin: d.tagMargins
```

Comment:
Why anchors? It's a `contentItem`

Reply:
```qml
        id: tagRow
        height: parent.height
        anchors.left: parent.left
        anchors.right: parent.right
        anchors.leftMargin: d.tagMargins
        anchors.rightMargin: d.tagMargins
```

all of this can be removed

Reply:
With the suggestion for `horizontalPadding: d.tagMargins`